src/components/Autocomplete/index.tsx

```diff
@@ -181,24 +181,7 @@ class AutocompleteInt<T> extends React.Component<AutocompleteProps<T> & FormCont
       }
     }
 
-    if (this._element) {
-      this._element.focus();
-    }
-
-    const hide = () => {
-      this.hide();
-      /*
-        On IE11 the focus event from this._element.focus(); triggers after this.hide() and so the suggestion list opens again after hiding it.
-        If we call this.hide() again with a timeout, it works also for IE11.
-      */
-      setTimeout(() => {
-        if (this.state.open) {
-          this.hide();
-        }
-      });
-    };
-
-    suggestionSelected ? hide() : this.show();
```

Comment:
unfortunately, it's not a solution. it will bring another issue https://github.com/ZEISS/precise-ui/issues/123

Reply:
see also https://github.com/ZEISS/precise-ui/pull/184